src/scrapers/visa-cal.js

```diff
@@ -190,25 +190,28 @@ async function getTransactionsForAllAccounts(authHeader, startMoment, options) {
     for (let i = 0; i < banksResponse.BankAccounts.length; i += 1) {
       const bank = banksResponse.BankAccounts[i];
       const bankDebits = await getBankDebits(authHeader, bank.AccountID);
-      if (_.get(bankDebits, 'Response.Status.Succeeded')) {
-        for (let j = 0; j < bank.Cards.length; j += 1) {
-          const rawTxns = await getTxnsOfCard(authHeader, bank.Cards[j], bankDebits.Debits);
-          if (rawTxns) {
-            let txns = convertTransactions(rawTxns);
-            txns = prepareTransactions(txns, startMoment, options.combineInstallments);
-            const result = {
-              accountNumber: bank.Cards[j].LastFourDigits,
-              txns,
-            };
-            accounts.push(result);
+      //Check that the bank has an active card to scrape
+      if (bank.Cards.some(_ => _.IsEffectiveInd)) {
```

Comment:
@ErezSavir looks much better :-)
My only comment now is about the usage of underscore `_`.
It's much nicer to see a relevant variable here. Besides, it's confusing due to the usage of underscore for lodash related operations.
So would be great if you could convert it to `bank.Cards.some(card => card.IsEffectiveInd)`.